jpsonic-main/src/main/java/com/tesshu/jpsonic/service/upnp/processor/ArtistByFolderUpnpProcessor.java

```diff
@@ -101,35 +99,22 @@ protected static final String createArtistId(String id) {
     public Container createContainer(FolderArtistAlbumWrapper item) {
         if (item.isArtist()) {
             MusicArtist container = new MusicArtist();
-            container.setId(getRootId() + UpnpProcessDispatcher.OBJECT_ID_SEPARATOR + item.getId());
-            container.setParentID(getRootId());
+            container.setId(getProcId().getValue() + ProcId.CID_SEPA + item.getId());
+            container.setParentID(getProcId().getValue());
             container.setTitle(item.getName());
             container.setChildCount(item.getArtist().getAlbumCount());
             if (item.getArtist().getCoverArtPath() != null) {
-                container.setProperties(Arrays.asList(
-                        new ALBUM_ART_URI(getDispatcher().getArtistProcessor().createArtistArtURI(item.getArtist()))));
+                container.addProperty(factory.toArtistArt(item.getArtist()));
             }
             return container;
```

Comment:
Check if it can be shared.

Reply:
Done. However, this problem will not be resolved until the music index is fixed. It will be completely standardized later.